scripts/generators/readmes.js

```diff
@@ -66,6 +66,7 @@ yarn add ${name} --dev
 
 packages
   .filter(x => x !== "README.md") // ignore root readme
+  .filter(x => x.indexOf("babel-preset-stage-") === -1) // ignore stages
```

Comment:
That's fine but just FYI this is a temporary change because we are going to not publish any more of the package anyway so I would remove this code the next release after.